apiary.gemspec

```diff
@@ -5,7 +5,7 @@ $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 require 'apiary/version'
 
 Gem::Specification.new do |gem|
-  gem.name          = 'apiaryio'
+  gem.name          = 'apiary'
```

Comment:
this is wrong `apiary` is another gem - https://rubygems.org/gems/apiary vs https://rubygems.org/gems/apiaryio